invenio_vocabularies/contrib/funders/schema.py

```diff
@@ -20,5 +27,21 @@ class FunderRelationSchema(Schema):
         required=True,
```

Comment:
minor: sorry didn't catch this earlier, shouldn't be required here (so we can set it to an "fk" funder)